Updated Neovim  to 0.5 version on windows and coc-flutter not working

I just installed Neovim 0.5.0 on windows and coc-flutter is not working. Then I installed Neovim 0.4.5 and its working fine now.
Any way it can work for Neovim 0.5.0

PS C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter> npm install

> coc-flutter@1.9.2 prepare
> husky install && npm run clean && npm run build

husky - Git hooks installed

> coc-flutter@1.9.2 clean
> rm -rf ./out

'rm' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code 1
npm ERR! path C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c husky install && npm run clean && npm run build

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jithu\AppData\Local\npm-cache\_logs\2021-07-22T08_47_28_285Z-debug.log
PS C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter> nvim C:\Users\jithu\AppData\Local\npm-cache\_logs\2021-07-22T08_47_28_285Z-debug.log

PS C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter> git reset --hard
HEAD is now at 12eb6e8 fix(version-windows): Fixed reading flutter version on windows
PS C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter> git pull --rebase
Already up to date.
PS C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter> npm install

> coc-flutter@1.9.2 prepare
> husky install && npm run clean && npm run build

husky - Git hooks installed

> coc-flutter@1.9.2 clean
> ./node_modules/.bin/rimraf ./out

'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code 1
npm ERR! path C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c husky install && npm run clean && npm run build

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jithu\AppData\Local\npm-cache\_logs\2021-07-22T11_24_20_208Z-debug.log
PS C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter>

PS C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter> npm install

> coc-flutter@1.9.2 prepare
> husky install && npm run clean && npm run build

husky - Git hooks installed

> coc-flutter@1.9.2 clean
> rimraf ./out


> coc-flutter@1.9.2 build
> NODE_ENV=production node esbuild.js

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code 1
npm ERR! path C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c husky install && npm run clean && npm run build

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jithu\AppData\Local\npm-cache\_logs\2021-07-22T11_32_48_862Z-debug.log
PS C:\Users\jithu\AppData\Local\nvim\plugged\coc-flutter>

--- a/CHANGELOD.md
+++ b/CHANGELOD.md
@@ -1,3 +1,7 @@
+# 1.9.3
+
+- Fixed #123 where sdk discovery on windows was not working properly.
+
 # 1.9.2
 
 - The FlutterDevices list now has new actions and also a new default.

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
     "coc": "^0.0.80"
   },
   "scripts": {
-    "clean": "./node_modules/.bin/rimraf ./out",
+    "clean": "rimraf ./out",
     "watch": "NODE_ENV=development node esbuild.js --watch",
     "build": "NODE_ENV=production node esbuild.js",
     "lint": "eslint src",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "coc-flutter",
-  "version": "1.9.2",
+  "version": "1.9.3",
   "description": "flutter support for (Neo)vim",
   "author": "iamcco <ooiss@qq.com>",
   "license": "MIT",

--- a/src/lib/sdk.ts
+++ b/src/lib/sdk.ts
@@ -1,5 +1,6 @@
 import { ExecOptions } from 'child_process';
 import { Uri, workspace, WorkspaceConfiguration } from 'coc.nvim';
+import { readFileSync } from 'fs';
 import os, { homedir } from 'os';
 import { dirname, join } from 'path';
 import which from 'which';
@@ -239,10 +240,19 @@ class FlutterSDK {
     return execCommand(`${this.flutterCommand} ${command}`, options);
   }
 
-  private async versionForSdk(location: string): Promise<string | undefined> {
-    const version = await execCommand(`cat ${join(location, 'version')}`);
-    if (version.err) return;
-    return version.stdout;
+  private versionForSdk(location: string): string | undefined {
+    const versionLocation = join(location, 'version');
+    if (!exists(versionLocation)) {
+      log(`${versionLocation} does not exist`);
+      return;
+    }
+    const version = readFileSync(versionLocation).toString().trim();
+    if (version.length == 0) {
+      log(`${versionLocation} was empty`);
+      return;
+    }
+    log(`${versionLocation} => ${version}`);
+    return version;
   }
 
   private async sdkWithLookup(flutterLookup: string, currentSdk: string): Promise<FlutterSdk | undefined> {
@@ -269,7 +279,7 @@ class FlutterSDK {
       }
       const isFlutterDir = exists(join(flutterPath, 'bin', 'flutter'));
       if (!isFlutterDir) return;
-      const version = await this.versionForSdk(flutterPath);
+      const version = this.versionForSdk(flutterPath);
       if (version) {
         return {
           location: flutterPath,
@@ -302,7 +312,7 @@ class FlutterSDK {
       path = path.replace(/^~/, home).trim();
       const isFlutterDir = exists(join(path, 'bin', 'flutter'));
       if (isFlutterDir) {
-        const version = await this.versionForSdk(path);
+        const version = this.versionForSdk(path);
         if (version && !sdks.some((sdk) => sdk.location == path)) {
           sdks.push({
             location: path,
@@ -320,7 +330,7 @@ class FlutterSDK {
         const isFlutterDir = exists(join(location, 'bin', 'flutter'));
         if (!isFlutterDir) continue;
         if (sdks.some((sdk) => sdk.location == location)) continue;
-        const version = await this.versionForSdk(location);
+        const version = this.versionForSdk(location);
         sdks.push({
           location: location,
           isFvm: false,
@@ -352,7 +362,7 @@ class FlutterSDK {
         const isFlutterDir = exists(join(location, 'bin', 'flutter'));
         if (!isFlutterDir) continue;
         if (sdks.some((sdk) => sdk.location == location)) continue;
-        const flutterVersion = await this.versionForSdk(location);
+        const flutterVersion = this.versionForSdk(location);
         sdks.push({
           location: location,
           fvmVersion: version,

